index.py

```diff
@@ -69,31 +74,36 @@ def main():
     if check_closing:
         closing_terms = has_closing_terms(pr_description)
         if not closing_terms:
-            errors.append("{}\n{}\n{}\n".format(
+            errors.append(no_closing_message or "{}\n{}\n{}\n".format(
                 "### ❌ Missing Closing Terms",
                 "This PR does not reference an issue with `closes`, `fixes`, or `resolves` keywords.",
                 "Please update the PR description to automatically close the relevant issue when merged."
             ))
   
     if check_boxes:
         [ is_not_closed, unclosed_boxes ] = has_unclosed_checkboxes(pr_description)
-        if is_not_closed:
-            res = ""
-            res += "### ❌ Unchecked Checkboxes\n"
-            res += "Some required checklist items in the PR description are not checked. Make sure all mandatory tasks are completed:\n"
+        if unclosed_boxes:
+            res = unchecked_boxes_message+"\n" or (
+                "### ❌ Unchecked Checkboxes\n"
+                "Some required checklist items in the PR description are not checked. "
+                "Make sure all mandatory tasks are completed:\n"
+            )+"\n"
```

Comment:
I'm not sure checking with unclosed_boxes here will work. It's a list. `is_not_closed` (boolean) makes more sense (what we had earlier)